scripts/zkai_d128_risc0_statement_receipt_gate.py

```
@@ -0,0 +1,618 @@
+    expect_equal(payload["decision"], DECISION, "decision")
+    expect_equal(payload["claim_boundary"], CLAIM_BOUNDARY, "claim boundary")
+    expect_equal(payload["route_id"], ROUTE_ID, "route id", layer="receipt_metadata")
+    expect_equal(payload["system"], SYSTEM, "system", layer="receipt_metadata")
+    expected_journal = expected_journal_contract()
+    expect_equal(payload["journal_contract"], expected_journal, "journal contract", layer="journal_contract")
+    expect_equal(payload["journal_commitment"], expected_journal["journal_commitment"], "journal commitment", layer="journal_contract")
+    artifact = require_object(payload["receipt_artifact"], "receipt artifact", layer="receipt_metadata")
+    expect_keys(artifact, {"path", "size_bytes", "sha256", "commitment"}, "receipt artifact", layer="receipt_metadata")
+    receipt_path = ROOT / artifact["path"]
+    if not receipt_path.is_file():
+        raise D128Risc0StatementReceiptError("receipt artifact missing", layer="receipt_artifact")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)